install.sh

```diff
@@ -16,9 +16,9 @@ init_db_final=0
 while [ "$init_db_valid" == 0 ]
 do
   read -p "Initialize database (y/n)? " init_db
-  case "${init_db,,}" in
-    y|yes ) init_db_valid=1; init_db_final=1;;
-    n|no ) init_db_valid=1; init_db_final=0;;
+  case "$init_db" in
+    [yY] ) init_db_valid=1; init_db_final=1;;
+    [nN] ) init_db_valid=1; init_db_final=0;;
     * ) echo "Invalid input" 1>&2;;
```

Comment:
This example seems a bit better with `[yY]*`, and a nicer error message:

https://stackoverflow.com/questions/226703/how-do-i-prompt-for-yes-no-cancel-input-in-a-linux-shell-script/226724#226724

Reply:
I updated it.